### ISSUE TITLE
Mic input is very low

### ISSUE THREAD

**Issue by OscarVegener** [SchmockLord/Gigabyte-Z590i-Vision-D-11900k#45]
Mic input is very low when connected to back panel of MB(well it's the only option with this MB). 
Also there are 2 usb input devices in Sound settings and only one of them is actually working. In Audio Midi Setup app, both devices have 2 ins and max db is 12 which is strange.
Does it have something to do with USB mapping?

My specs:
MB: Gigabyte Z590i Vision D
CPU: i5 10400f
GPU: RX 6600 XT
RAM: 2x8 CL18 3600 Mhz(no issues at all running at 3600 Mhz)

**Comment by SchmockLord** [comment]
Yeah I know about this issue. This board has Realtek 1220. Fortunately this guide is with the same chipset as example: https://dortania.github.io/OpenCore-Post-Install/universal/audio.html#finding-your-layout-id


I will try this but I am not sure if it is really working with AppleALC.kext, because it is via USB in our boards case.

**Comment by SchmockLord** [comment]
So I tried it again with AppleALC.kext and ALCID=11. No difference in the recording. I also didn't expect that AppleALC.kext is working with the Realtek 1220 connected via USB.

But my recording was ok. Of course, not the loudest, but totally fine for a integrated motherboard audio.

**Comment by OscarVegener** [comment]
@SchmockLord Thanks, I will try AppleALC.kext too.
